reports/report_details.py

```diff
@@ -1527,7 +1527,8 @@ def add_transnational_to_regions(regions):
             'name': '98',
             'title': 'Twitter - Images',
             'desc': '',
-            'reports': [],
+            'historical': '98',
+            'reports': ['global'],
```

Comment:
Why are we changing this? It's not related to this query.

Reply:
That was the first query I tried working on when I created this PR, that's why I made this change. I'll update to revert the change. @kilemensi